src/ui/public/stringify/types/source.js

```diff
@@ -1,23 +1,26 @@
 import _ from 'lodash';
 import { noWhiteSpace } from 'ui/utils/no_white_space';
-import angular from 'angular';
-import { IndexPatternsFieldFormatProvider } from 'ui/index_patterns/_field_format/field_format';
+import { toJson } from 'ui/utils/aggressive_parse';
+import { FieldFormat } from 'ui/index_patterns/_field_format/field_format';
 
-export function stringifySource(Private, shortDotsFilter) {
-  const FieldFormat = Private(IndexPatternsFieldFormatProvider);
+export function stringifySource() {
   const template = _.template(noWhiteSpace(require('ui/stringify/types/_source.html')));
 
   _.class(Source).inherits(FieldFormat);
-  function Source(params) {
+  function Source(params, getConfig) {
     Source.Super.call(this, params);
+
+    this.getConfig = getConfig;
   }
 
   Source.id = '_source';
   Source.title = '_source';
   Source.fieldType = '_source';
 
   Source.prototype._convert = {
-    text: angular.toJson,
+    text: (value) => {
+      return toJson(value);
```

Comment:
why this change? If relevant, maybe we should add a comment that explains it?

Also, this can be: `text: (value) => toJson(value)`. No need for `{}` here.